common/build/eslint-config-fluid/package.json

```diff
@@ -28,6 +28,7 @@
 		"test": "echo TODO: add tests"
 	},
 	"dependencies": {
+		"@fluid-internal/eslint-plugin-fluid": "^0.1.1",
```

Comment:
Note 

`0.1.1` will be available once https://github.com/microsoft/FluidFramework/pull/19900 is merged.